assets/pages/paper-subscription-landing/paperSubscriptionLandingPageActions.js

```diff
@@ -0,0 +1,25 @@
+// @flow
+
+// ----- Imports ----- //
+
+import { type PaperBillingPeriod } from 'helpers/subscriptions';
+import { sendTrackingEventsOnClick } from 'helpers/subscriptions';
+import { productPagePeriodFormActionsFor } from 'components/productPage/productPagePeriodForm/productPagePeriodFormActions';
+
+// ----- Action Creators ----- //
+
+const { setPeriod } = productPagePeriodFormActionsFor<PaperBillingPeriod>('Paper', 'Paper');
+
+function redirectToWeeklyPage() {
+  const location = null;
+
+  if (location) {
```

Comment:
would this ever pass, given `location = null` above?

Reply:
ugh, this whole method should be taken out actually, it's a horrible cp from the GW page